src/App.js

```diff
@@ -89,14 +91,41 @@ function App() {
     setCardsText([...newCards]);
   };
 
+  const addToSelectionListHandler = (cardId, isSelected, isEdited) => {
+    //adding selected cards to the list
+    if (isSelected && !isEdited) {
+      setSelectionList((prevList) => [...prevList, cardId]);
+      //deleting cards from the list if they were unselected
+    } else if (!isSelected) {
+      let index = selectionList.indexOf(cardId);
+
+      if (index !== -1) {
+        let tempList = [...selectionList];
+        tempList.splice(index, 1);
+        setSelectionList(() => [...tempList]);
+      }
+    }
+  };
+
+  //clicking the delete button
+  const deleteCardsHandler = () => {
+    let tempCards = [...cardsText];
+    tempCards = tempCards.filter((card) => !selectionList.includes(card.id));
+    setCardsText([...tempCards]);
```

Comment:
This is OK, but can actually be done as one line statement. However in terms of readability, you are free to leave as is.